core/templates/dev/head/app.js

```diff
@@ -93,6 +93,12 @@ oppia.config(['$provide', function($provide) {
   }]);
 }]);
 
+//Returns true if the user is on a mobile device.
+//See here: http://stackoverflow.com/a/14301832/5020618
+oppia.factory('deviceInfoService', ['$window', function($window) {
+  return typeof $window.orientation !== 'undefined';
```

Comment:
Actually I just realized that this should return (like other services) a JS object containing a single key-value pair. The key is 'isMobileDevice' and the value is a function that returns a boolean.

This allows to add more functions later if we wanted to know more about the device.